front/lib/connectors_api.ts

```diff
@@ -49,56 +49,43 @@ export const ConnectorsAPI = {
   },
 
   async pauseConnector(
-    provider: ConnectorProvider,
-    workspaceId: string,
-    dataSourceName: string
+    connectorId: string
   ): Promise<ConnectorsAPIResponse<{ connectorId: string }>> {
-    const res = await fetch(`${CONNECTORS_API}/connectors/pause/${provider}`, {
-      method: "POST",
-      headers: getDefaultHeaders(),
-      body: JSON.stringify({
-        workspaceId,
-        dataSourceName,
-      }),
-    });
+    const res = await fetch(
+      `${CONNECTORS_API}/connectors/pause/${connectorId}`,
+      {
+        method: "POST",
+        headers: getDefaultHeaders(),
+      }
+    );
 
     return _resultFromResponse(res);
   },
 
   async resumeConnector(
-    provider: ConnectorProvider,
-    workspaceId: string,
-    workspaceAPIKey: string,
-    dataSourceName: string,
-    nangoConnectionId: string
+    connectorId: string
```

Comment:
I am surprised to see that nobody is calling this function in our code base. Is that expected @fontanierh ?

Reply:
yes you're right the endpoint isn't used, but it was very low effort to add since we already have an endpoint for every other action